training_configs/vilbert_ve_from_huggingface.jsonnet

```diff
@@ -0,0 +1,74 @@
+local model_name = "bert-base-uncased";
+local effective_batch_size = 128;
+local gpu_batch_size = 32;
+local num_gpus = 4;
+
+local datadir = "/net/s3/allennlp/akshitab/data/SNLI-VE/data/";
+
+{
+  "dataset_reader": {
+    "type": "visual-entailment",
+    "image_dir": datadir + "Flickr30K/flickr30k_images",
+    "feature_cache_dir": datadir + "/feature_cache",
+    "image_loader": "detectron",
+    "image_featurizer": "resnet_backbone",
+    "region_detector": "faster_rcnn",
+    "tokenizer": {
+      "type": "pretrained_transformer",
+      "model_name": model_name
+    },
+    "token_indexers": {
+      "tokens": {
+        "type": "pretrained_transformer",
+        "model_name": model_name
+      }
+    },
+    "max_instances": 30000,
+    "image_processing_batch_size": 16,
+  },
+  "validation_dataset_reader": self.dataset_reader,
```

Comment:
If you don't modify the reader for validation, you can just omit the whole line.